Cargo.toml

```diff
@@ -109,9 +109,9 @@ optional = true
 version = "0.8"
 features = ["v4"]
 
-[dependencies.xsalsa20poly1305]
+[dependencies.crypto_secretbox]
```

Comment:
Can you move the `[dependencies.crypto_secretbox]` block to its correct alphabetical place? I.e., between byteorder and dashmap.